ops/testing.py

```diff
@@ -1495,7 +1495,13 @@ def push(
 
     def list_files(self, path: str, *, pattern: str = None,
                    itself: bool = False) -> typing.List[pebble.FileInfo]:
-        files = [self._fs.get_path(path)]
+        try:
+            files = [self._fs.get_path(path)]
+        except FileNotFoundError as e:
+            # conform with the real pebble api
+            raise pebble.APIError(
+                body={}, code=404, status='Not Found', message=str(e))
```

Comment:
Would it be worth setting the APIError message equal to what we would see from pebble?  From digging through pebble code - it looks like the error/message pebble provides will be: `open /path/that/does/not/exist: no such file or directory`.

Reply:
Agreed. Not a must as people shouldn't be testing error strings, but doesn't hurt to be consistent either.

Reply:
It was my intention to keep it consistent but I couldn't verify the exact error wording. Thanks, will do that!